lib/libutils/isoc/bget_malloc.c

```diff
@@ -609,7 +613,11 @@ void *mdbg_malloc(const char *fname, int lineno, size_t size)
 {
 	struct mdbg_hdr *hdr;
 
+#if defined(ARM64)
+	COMPILE_TIME_ASSERT(sizeof(struct mdbg_hdr) == sizeof(uint32_t) * 6);
```

Comment:
Hmm, I wish I'd have left a comment here on why we have this assert.
I think we want to check that the `struct mdbg_hdr` doesn't give bad alignment.
Please change this compile time assert to check that `(sizeof(struct mdbg_hdr) % __alignof(uintptr_t)) == 0` or similar and add a comment on what we're testing.